src/evm/EvmDiamondStateMachine.ts

```diff
@@ -376,13 +378,21 @@ class EvmDiamondStateMachine extends ADiamondStateMachine {
         // Get time configuration
         const configTimes =
             await deployedStateChannelContractInstance.getAllTimes();
-        const timeConfig: TimeConfig = {
+        let timeConfig: TimeConfig = {
             p2pTime: Number(configTimes[0]),
             agreementTime: Number(configTimes[1]),
             chainFallbackTime: Number(configTimes[2]),
             evidenceTime: Number(configTimes[3])
         };
 
+        // Apply time configuration override if provided (for testing)
+        if (timeConfigOverride) {
+            timeConfig = {
+                ...timeConfig,
+                ...timeConfigOverride
+            };
```

Comment:
btw you know that you can just define these values in the test contract (MathStateMachine) and that you don't need to override anything - the values in the contract will be fetched